lrauv_ignition_plugins/src/WorldCommPlugin.cc

```diff
@@ -104,15 +105,22 @@ void WorldCommPlugin::Configure(
   this->createService = "/world/" + topicWorldName + "/create";
   this->setSphericalCoordsService = "/world/" + topicWorldName
     + "/set_spherical_coordinates";
+
+  // Advertise service for world origin in spherical coordinates
+  this->getWorldOriginSphericalService = "/world_origin_spherical";
+  this->node.Advertise(this->getWorldOriginSphericalService,
+    &WorldCommPlugin::WorldOriginSphericalService, this);
```

Comment:
See https://github.com/osrf/lrauv/pull/77 for a way to detect that the spherical coordinates changed without a service.

Reply:
Thank you! Merged so that I can update this PR to it

Reply:
Incorporated into this PR 3f6edfc

Reply:
I saw you resolved this comment; are you planning on removing this service? I think it isn't needed, right?

Reply:
Oh I forgot to remove it in WorldCommPlugin! Oops thanks for keeping track. Removed in 58af8e8
PR ready for another pass. I rebased my interpolation branch on this PR, so it'd be nice to get this one in.